packages/@react-spectrum/s2/src/ProgressCircle.tsx

```diff
@@ -54,17 +54,52 @@ const track = style({
     default: 'gray-300',
     isStaticColor: 'transparent-overlay-300',
     forcedColors: 'Background'
+  },
+  strokeWidth: {
```

Comment:
Can we keep the stroke with in the JS instead of in multiple places? Did it need to move into the style macro for some reason?

Reply:
yes, I moved it there so I could do high contrast mode